### ISSUE TITLE
Add a beautiful logo

### ISSUE THREAD

**Issue by ppamorim** [mrmans0n/smart-location-lib#104]
It would be amazing to add a logo and another visual ways to makes this library more interactive. What do you think about?

**Comment by kozmic** [comment]
I agree, a great library like this should have a logo. Here is my contribution ;)
![smart_location_lib_logo](https://cloud.githubusercontent.com/assets/6666/12713463/2ba7085a-c8cf-11e5-9c94-9eaa129608d2.png)

**Comment by mrmans0n** [comment]
That is a really beautiful logo :+1:

**Comment by ekimia** [comment]
+1

**Comment by Max01010101010101** [comment]
This should be oficial!! +1
